common/persistence/domainReplicationQueue.go

```diff
@@ -137,6 +147,10 @@ func (q *domainReplicationQueueImpl) purgeAckedMessages() error {
 		return fmt.Errorf("failed to purge messages: %v", err)
 	}
 
+	q.metricsClient.
+		Scope(metrics.FrontendDomainReplicationQueueScope, metrics.SourceClusterTag(q.clusterName)).
```

Comment:
i believe the source cluster tag is not necessary

Reply:
You are right. I'll remove.